suite-native/atoms/src/Sheet/BottomSheetFlashList.tsx

```diff
@@ -0,0 +1,146 @@
+import { useEffect, useRef, ReactNode } from 'react';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+import Animated from 'react-native-reanimated';
+import { PanGestureHandler } from 'react-native-gesture-handler';
+import { GestureResponderEvent, Pressable } from 'react-native';
+
+import { FlashList, FlashListProps } from '@shopify/flash-list';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { Box } from '../Box';
+import { BottomSheetContainer } from './BottomSheetContainer';
+import { useBottomSheetAnimation } from './useBottomSheetAnimation';
+import { BottomSheetHeader } from './BottomSheetHeader';
+
+export type BottomSheetFlashListProps<TItem> = {
+    isVisible: boolean;
+    isCloseDisplayed?: boolean;
+    onClose: (isVisible: boolean) => void;
+    title?: ReactNode;
+    subtitle?: ReactNode;
+    ExtraProvider?: React.ComponentType;
+    estimatedListHeight?: number;
+} & FlashListProps<TItem>;
+
+const DEFAULT_INSET_BOTTOM = 50;
+
+const sheetWrapperStyle = prepareNativeStyle(utils => ({
+    backgroundColor: utils.colors.backgroundSurfaceElevation0,
+    borderTopLeftRadius: utils.borders.radii.large,
+    borderTopRightRadius: utils.borders.radii.large,
```

Comment:
This is obsolete, should use `r20` after rebasing.
```suggestion
    borderTopLeftRadius: utils.borders.radii.r20,
    borderTopRightRadius: utils.borders.radii.r20,
```